pages/osx/split.md

```diff
@@ -6,9 +6,9 @@
 
 `split -l {{10}} {{filename}}`
 
-- Split a file into 5 files. File is split such that each split has same size (except the last split):
+- Split a file by a regular expression. The matching line will be the first line of the next output file:
 
-`split -n {{5}} {{filename}}`
+`split -p {{regular expression}} {{filename}}`
```

Comment:
A concrete example of a regex would be helpful here.

Reply:
should `{{filename}}` be like `{{animals.txt}}`?